openapi/pagopa-arc-be.openapi.yaml

```diff
@@ -121,6 +121,31 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ErrorDTO'
+  /auth/testuser:
+    get:
+      tags:
+        - arc auth
+      summary: "Provide access token for the sample user"
+      operationId: getTestUserInfo
```

Comment:
not really a 'userInfo', may we found a more suitable name?

Reply:
renamed